openapi-v1.yaml

```diff
@@ -2898,6 +2898,34 @@ definitions:
         x-omitempty: true
         description: License text
 
+  GroupMember:
+    description: A groups member, array or another groups, to add or remove from an existing group.
+    type: object
+    properties:
+      namespace:
+        description: The namespace of the asset.
+        type: string
+      name:
+        description: The name or id of the asset.
+        type: string
+      asset_type:
+        $ref: '#/definitions/AssetType'
+
+  GroupChanges:
+    description: A request to change the members of a group. Contains assets to add or remove.
+    type: object
+    properties:
+      add:
```

Comment:
I don't understand the use of this type.
Shouldn't we use a POST and a DELETE method with the same body (GroupMember)?

Reply:
For this case, we think that a single batch operation aligns better with the upcoming core groups implementation

Reply:
Delete bodies are not universally supported either